### PR TITLE
set password expiry to zero when upgrading

### PR DESCRIPTION
**Pull request by zoltankocsardi**
## Description
Drung the upgrade of SuiteCRM, password expiry's set to zero and user be informed about that.

## How To Test This
- upgrade Suite
- check the warning message about password expiry in the end.
- check password expiry changed in password management [or systexpiration => '' in config.php[]

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by pgorod** [comment]
@zoltankocsardi shouldn't that new message be a translatable string? Does it show on screen, or only in the logs?

**Comment by pgorod** [comment]
Sorry, didn't realize it was WIP. Thanks!

File: modules/UpgradeWizard/language/en_us.lang.php [hunk 1]
```diff
@@ -249,6 +249,7 @@
     'ERROR_VERSION_INCOMPATIBLE' => 'The uploaded file is not compatible with this version of SuiteCRM: ',
     'ERROR_PHP_VERSION_INCOMPATIBLE' => 'The uploaded file is not compatible with this version of PHP: ',
     'ERROR_SUITECRM_VERSION_INCOMPATIBLE' => 'The uploaded file is not compatible with this version of SuiteCRM: ',
+    'ERROR_FLAVOR_INCOMPATIBLE' => 'The uploaded file is not compatible with this flavor (Community Edition, Professional, or Enterprise) of SuiteCRM: ',
```
Comment (pgorod) [inline modules/UpgradeWizard/language/en_us.lang.php]: We have flavors of SuiteCRM? We have paid versions? 😱

Comment (zoltankocsardi) [reply modules/UpgradeWizard/language/en_us.lang.php]: Tbh I haven't noticed this. The upgrade script did that.
@mattlorimer ?

Comment (horus68) [reply modules/UpgradeWizard/language/en_us.lang.php]: That string was removed here: https://github.com/salesagility/SuiteCRM/pull/5482
from:
install/language/en_us.lang.php
modules/UpgradeWizard/language/en_us.lang.php

Comment (zoltankocsardi) [reply modules/UpgradeWizard/language/en_us.lang.php]: It seems I've used an older upgrade pack for testing [although I was 7.10], but now it's fixed.
Thank you guys!

**Comment by Dillon-Brown** [comment]
Assessed

**Pull request merged**